docs/docs/overview/does-testkube-replace-cicd.md

```diff
@@ -0,0 +1,9 @@
+# Does Testkube replace my CI/CD?
+
+The beauty of Testkube is that it improves your CI/CD pipeline!
+
+There's different ways to integrate Testkube with your current CI/CD. You can directly use the CLI, or if you use GitHub, you can create GitHub Actions. Read more about the process [here](https://kubeshop.github.io/testkube/integrations/testkube-automation).
```

Comment:
There are different ways to integrate Testkube with your current CI/CD. You can directly use the CLI, or, if you use GitHub, you can create GitHub Actions. Read more about the process [here](https://kubeshop.github.io/testkube/integrations/testkube-automation).